src/clj_http/conn_mgr.clj

```diff
@@ -274,7 +274,7 @@
 
 (defn shutdown-manager
   "Shut down the given connection manager, if it is not nil"
-  [manager]
+  [^org.apache.http.conn.HttpClientConnectionManager manager]
```

Comment:
Stand by - this line is incorrect.  It should be more like the new `shutdown` function I added in [`clj-http.core`](https://github.com/dakrone/clj-http/pull/358/files#diff-1d42f09b3425eeaeda024dcdc7d49145R332).